00-programming-fundamentals/01-how-to-osx.md

```diff
@@ -1,55 +1,150 @@
+Command-/ | `⌘/` | Make either the current line or a selection of text a comment (Sublime or Atom only)
+Command-shift-f | `⌘⇧f` | Fullscreen
+
+
+## A Word About Mice
+
+Steve Jobs had a [weird thing](http://www.cultofmac.com/269222/steve-jobs-hated-idea-multi-button-mouse-designer-claims/) about single-button mice.  Trust me, it's bizarre.  Over time Apple's added 2-button mouse functionality to MacOS, mostly to open rarely accessed sub-menus, and you'll be surprised how little you actually use it, if you think about it at all.  You can make a right click on your laptop trackpad by either by clicking and holding the option key down, or pressing with two fingers on the trackpad.  Or... you can just plug in a regular PC mouse and it'll work fine.  
+
+
+## Using a Windows Keyboard
+
+
+If you do use an external windows keyboard & mouse at home you'll find a few differences.  Most notably the mac will consider the "windows" key the command key.  Below is a table listing windows keys and how the Mac interprets them.  
+
+ 
+Windows Key | Mac Key 
+--------|:----------
+Control | Control!
+Windows | Command
+Alt     | Alt/Option
+Escape  | Cancels out of active window / element or cancels next character
+Insert  | Opens the *help* dialog
+Delete  | Delete :p
+Page up & Page Down | Moves the window up or down but does **not** move the cursor. 
+Home/End | Jumps to the bottom of the document but does **not** move the cursor. 
+Windows + Home/End | Jumps **with** cursor to top/bottom of document
+Numlock | Nothing, nada
+10-key-pad | Also nothing
+
+
+
+
+
+## Taking Screenshots
+
+Screenshots on a Mac are very cool. By default, they save a file to the desktop. If you add Control (⌃) to the shortcut, the screenshot will be copied to the clipboard instead.
+
+
+Behavior / PC Key | Mac Keyboard Shortcut | In Symbols!
+------------------|-----------------------|------------
+Print Screen (full screen, saved to desktop) | Command + Shift + 3 | ⌘⇧3 (⌘#)
+Print Screen (full screen, copied to clipboard) | Control + Command + Shift + 3 | ⌃⌘⇧3 (⌃⌘#)
+Print Screen (select area, saved to desktop) | Command + Shift + 4 | ⌘⇧4 (⌘$)
+Print Screen (select area, copied to clipboard) | Control + Command + Shift + 4 | ⌃⌘⇧4 (⌃⌘$)
+Print Screen (application window, PC: Alt + Print Screen) | Command + Shift + 4, Space-bar, Click on Window | ⌘⇧4 (⌘$), {Space}, Click on Window
+
+
+<a id="#custom-shortcuts"></a>
+
+## Custom Shortcuts
+
+
+There might be applications that will do some or all of this for you, but I opted to make the changes manually. Some function keys were easier to reprogram than others. You'll need to open __System Preferences__ and go into __Keyboard__ settings. In the __Shortcuts__ tab, check out your options in the list on the lift. It should say things like __Launchpad & Dock__, __Mission Control__, and __Keyboard__. Click on __App Shortcuts__ at the bottom. Now you can customize your own shortcuts!
```

Comment:
I think we should clean up this section to be more like "here is how you add shortcuts" instead of the specific opinions of the writer